src/Workspaces/Core/Portable/Options/GlobalOptionService.cs

```diff
@@ -14,259 +14,282 @@
 using Microsoft.CodeAnalysis.Shared.Utilities;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Options
+namespace Microsoft.CodeAnalysis.Options;
```

Comment:
In the future, please make the change to file-scoped namespaces as its own separate commit. If combined with another PR, please do not squash merge it.